cvat-sdk/cvat_sdk/core/downloading.py

```diff
@@ -27,21 +29,52 @@ class Downloader:
     def __init__(self, client: Client):
         self._client = client
 
+    @staticmethod
+    def _get_server_filename(response: urllib3.HTTPResponse) -> str:
+        content_disposition = next(
+            (
+                filename
+                for part in response.headers.get("Content-Disposition", "").split(";")
+                if (filename := part.strip()) and filename.startswith("filename=")
```

Comment:
The code is a bit confusing, because here `filename` includes the `filename=` string, but below it doesn't. Maybe you could rename this to `filename_parameter` or something?

Reply:
Ok, renamed